README.md

```diff
@@ -0,0 +1,33 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+In this project I refactored the previous repo "awesome-books" to use ES6. I build a aplication called Awesome Books where you can create a list of your favorites books, this aplication was built with plenty JavaScript to add and remove elements from the DOM and store data in the local storage.
+
```

Comment:
Please ensure to have your `README.md` file updated to make your project look professional. Please the `Awesome Books` in to `To-Do-List`

Reply:
Hello @ABDULALI3468 you have not made the changes requested by the previous reviewer, is there any problem?